kof-operator/internal/controller/clusterdeployment_kof_cluster_role.go

```diff
@@ -171,6 +179,68 @@ func (r *ClusterDeploymentReconciler) reconcileChildClusterRole(
 	return nil
 }
 
+func (r *ClusterDeploymentReconciler) createProfile(childClusterDeployment, regionalClusterDeployment *kcmv1alpha1.ClusterDeployment, ctx context.Context) error {
+	log := log.FromContext(ctx)
+	remoteSecretName := istio.RemoteSecretNameFromClusterName(regionalClusterDeployment.Name)
+
+	log.Info("Creating profile")
+
+	profile := &sveltosv1beta1.Profile{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      remoteSecretName,
+			Namespace: childClusterDeployment.Namespace,
+			Labels: map[string]string{
+				"app.kubernetes.io/managed-by": "kof-operator",
+			},
+			OwnerReferences: []metav1.OwnerReference{
+				GetOwnerReference(
+					childClusterDeployment.Name,
+					childClusterDeployment.GetUID(),
+				),
```

Comment:
Part 3 of DRY-ing it:
```suggestion
				GetOwnerReference(childClusterDeployment),
```